common/terms.html

```diff
@@ -53,10 +53,22 @@
     <dd>
       <p>In this specification, attribute is used as it is in markup languages. Attributes are structural features added to <a class="termref" href="#dfn-element">elements</a> to provide information about the <a class="termref" href="#dfn-state">states</a> and <a class="termref" href="#dfn-property">properties</a> of the <a class="termref" href="#dfn-object">object</a> represented by the element.</p>
     </dd>
+    <dt><dfn data-lt="child element|child|children|child elements">Child Element</dfn></dt>
+    <dd>
+      <p>An 'child element' is a direct <abbr title="Document Object Model">DOM</abbr> child of the <a>element</a>, an descendant of the <a>element</a> with only <rref>generic</rref> or <rref>presentation</rref> elements intervening, or a child specified via <pref>aria-owns</pref> on the element.</p>
```

Comment:
I find it a little confusing to use "element" here, since a child might be text, which is not an element.
And I find "child element" to be confusing because it contradicts what a child element means in the DOM, given that aria-owns children are not child elements.

In Chrome, we would avoid saying "element" unless something is truly an element, and would avoid saying child element if something is not actually a DOM child element. We might say child or child object. Maybe we want to say owned child here?

Reply:
I'll updated with "owned child", and I'll also change "Allowed Child Elements" to "Allowed Child Roles" both to match the "Required Context Role" and to avoid element 👍